docs/docs/self-host/customize-deployment/use-sso-login-for-self-hosted-lightdash.mdx

```diff
@@ -65,13 +65,25 @@ server has an issuer URI of `https://dev-123456.okta.com/oauth2/default`.
 
 You'll need to set the following environment variables in your Lightdash deployment:
 
-| Variable                            | Description                                                                                        | Required? |
-|-------------------------------------|----------------------------------------------------------------------------------------------------|-----------|
+| Variable                            | Description                                                                                        | Required?  |
+|-------------------------------------|----------------------------------------------------------------------------------------------------|------------|
 | `AUTH_OKTA_DOMAIN`                  | The `{{ okta_domain }}`. Should not include `https://`                                             | ✅         |
 | `AUTH_OKTA_OAUTH_CLIENT_ID`         | The **Client ID** copied from the application settings in okta                                     | ✅         |
 | `AUTH_OKTA_OAUTH_CLIENT_SECRET`     | The **Client secret** copied from the application settings in okta                                 | ✅         |
 | `AUTH_OKTA_OAUTH_ISSUER`            | The **Issuer URI** copied from the authorization server. Should include `https://`                 | ✅         |
-| `AUTH_OKTA_AUTHORIZATION_SERVER_ID` | Optional. The **Name** of a custom authorization server if not using the org authorization server. |           |
+| `AUTH_OKTA_AUTHORIZATION_SERVER_ID` | Optional. The **Name** of a custom authorization server if not using the org authorization server. |            |
+| `AUTH_OKTA_EXTRA_SCOPES`            | Optional. The extra **scopes** (e.g. "groups") when not using a custom authorization server        |            |
+
+#### Groups & Okta
```

Comment:
Could you move this above the table @joaoviana ? The structure of the doc is: 

1. Okta configuration comes first
2. Lightdash env vars come afterward, once Okta is configured

Reply:
absolutely, thanks for pointing this out!